main.tf

```diff
@@ -34,7 +35,10 @@ resource "aws_ecs_service" "service" {
     }
   }
 
-  tags = var.tags
+  triggers = {
```

Comment:
This is hardcoded when it should be a dynamic field, you can then use the `timestamp()` instead on the example.